src/interfaces/common.ts

```diff
@@ -37,6 +37,7 @@ declare global {
 			session: {
 				jiraHost?: string;
 				githubToken?: string;
+				gitHubUuid?: string;
```

Comment:
/nit or JFYI,

Personally I'd prefer to carry the `uuid` to different pages via url (`/github/uuid`), instead put it in the session token.

But can leave it as it is.